particle-iot/particle-cli
# Put a Protected Device automatically in Service Mode to run CLI commands

This is the base PR.

This PR adds a helper module `device-protection-helper.js` that can be used by multiple modules and prevent circular dependency issues. It also adds a new method that checks if a device is ready for use (ie rebooted after a certain reset operation was done)

## How to Test

See this [test plan](https://www.notion.so/particle/Test-Plan-for-CLI-commands-Protected-Open-23954df3eff94cad9b3002722f926fa9)

## Related Issues / Discussions




## Completeness
